### PR TITLE
Add context-sensitive help molecule

### PR DESCRIPTION
**Pull request by dmarcoux**

### CODE REVIEW

**Comment by hennevogel** [comment]
Deploy preview for *obs-patterns* ready!

Built with commit a5175b30f3e59ba8608a776274d0390968e6af81

https://deploy-preview-64--obs-patterns.netlify.com

File: _posts/molecules/2019-08-08-tooltips.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+layout: post
+title: Tooltips
```
Comment (hennevogel) [inline _posts/molecules/2019-08-08-tooltips.html]: Isn't this more like `Help` or `Context sensitive help`?

Comment (dmarcoux) [reply _posts/molecules/2019-08-08-tooltips.html]: I first thought `Legend` would be nice, then went for `Tooltips`, but it's true that it's way too generic. I like `Context sensitive help`.

**Comment by dmarcoux** [comment]
@hennevogel Please have another look.

**Review by hennevogel — APPROVED** [review]

**Pull request merged**